src/System.Text.Primitive/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("HP Inc.")]
```

Comment:
...I honestly have no idea. Is that a default for AssemblyInfo.cs? Because I didn't touch that file hahaha. In any case, changing now.